mivs/templates/emails/game_submitted.txt

```diff
@@ -0,0 +1,7 @@
+{{ game.studio.primary_contact.first_name }},
```

Comment:
These automated emails are just placeholders.  After I'm provided with the real text that is wanted, it'll be easy to add.